<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item row="0" column="1">
    <widget class="QToolBox" name="toolBox">
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="domainPage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>250</width>
        <height>514</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Диаграмма предметной области</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="addRectangleButton">
         <property name="text">
          <string>Прямоугольник</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addLineButton">
         <property name="text">
          <string>Линия</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteItemButton">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadBinaryButton">
         <property name="text">
          <string>Открыть</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveBinaryButton">
         <property name="text">
          <string>Сохранить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveAsPictureButton">
         <property name="text">
          <string>Сохранить как картинку</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="usecasePage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>250</width>
        <height>514</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Диаграмма прецедентов</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="addActorButton">
         <property name="text">
          <string>Актер</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addLinkButton">
         <property name="text">
          <string>Связь</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadUsecaseBibaryButton">
         <property name="text">
          <string>Открыть</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveUsecaseBinaryButton">
         <property name="text">
          <string>Сохранить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveUsecaseAsPictureButton">
         <property name="text">
          <string>Сохранить как картинку</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="usecaseBlocksPage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>250</width>
        <height>514</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Диаграмма прецедентов: блок-схема</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
